src/PrestaShopBundle/Entity/Repository/NormalizeFieldTrait.php

```diff
@@ -48,6 +48,25 @@ protected function castNumericToInt($rows)
     }
 
     /**
+     * @param $rows
+     * @return mixed
+     */
+    protected function castIdsToArray($rows)
+    {
+        $castIdentifiersToArray = function (&$columnValue, $columnName) {
+            if ($this->shouldCastToInt($columnName)) {
+                $columnValue = explode(',', $columnValue);
+            }
+        };
+
+        array_walk($rows, function (&$rowColumns) use ($castIdentifiersToArray) {
```

Comment:
I just take the same logic as https://github.com/PrestaShop/PrestaShop/pull/8068/files#diff-112821fa3e3c8836d263160a9725a469L35, WDYT ?